### DISCUSSION TITLE
Where to get IBXC test tokens?

### DISCUSSION THREAD

**Discussion by zesushv** [IBAX-io/go-ibax#978] [Q&A] [answered]
Is there like a faucet to get IBXC tokens to try out the blockchain?

**Comment by ibaxio** [comment, marked as answer]
IBXC is IBAX Coin and it is not any kinds of Token.  Users can create their own ECOsystem and emission their ECO token, even make token as GAS Fee.

Right now when you visit https://weaver.ibax.io/ and sign in a new account that you will get 10,000 IBXC of TESTNET. You could use these TESTNET IBXC to get more experiences. Please check the explorer of IBAX Network: https://scan.ibax.io/

Thanks!
